lib/src/models/Cell.dart

```diff
@@ -5,7 +5,13 @@ import 'ICell.dart';
 import 'Position.dart';
 
 /// Represents a single cell in a 9x9 grid
-class Cell extends ICell {
+class Cell{
```

Comment:
```suggestion
class Cell extends ICell {
```

_Part of the 'let's keep ICell' movement ;)_